src/Apps/Artist/routes.tsx

```diff
@@ -226,6 +232,42 @@ export const routes: RouteConfig[] = [
       },
 
       // Routes not in tabs
+
+      {
+        path: "consign",
```

Comment:
I guess I was expecting to see the redirection happening here, but maybe this is the wrong place to check the list of artist names? Like, sorta weird to load up that data here instead of inside the component, is that the idea?

Reply:
The problem is that at this level we don't get access to `admin` properties. Once we're ready to deploy we can move the redirect up a level to here.

Reply:
Cool, sounds good! 👍